metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -203,36 +204,48 @@ def list_runs(page_offset: int, page_size: int, include_soft_deletes: bool) -> N
 
 @ingest.command()
 @click.option("--run-id", required=True, type=str)
+@click.option(
+    "--start",
+    type=int,
+    default=0
+)
+@click.option(
+    "--count",
+    type=int,
+    default=100
+)
+@click.option(
+    "--include-soft-deletes",
+    is_flag=True,
+    default=False,
+    help="If enabled, will list ingestion runs which have been soft deleted",
```

Comment:
shouldn't this say "if enabled, will show soft deletions that occurred in the ingestion run"?

Reply:
Fixed!